test/index.test.ts

```diff
@@ -130,17 +126,12 @@ describe('SCHEMA', () => {
                 type: '#33:2',
                 createdBy: '#44:1',
                 updatedBy: '#44:1',
-                break1Start: { '@class': 'ProteinPosition', pos: 1, refAA: 'A' },
+                break1Start: {
+                    '@class': 'ProteinPosition', pos: 1, refAA: 'A', prefix: 'p',
+                },
                 break1Repr: 'bad',
             }, { addDefaults: true });
             expect(formatted).toHaveProperty('break1Repr', 'p.A1');
         });
     });
-
-    describe('previews and identifiers', () => {
```

Comment:
why did you delete this test?

Reply:
Because identifiers is a property that we no longer add/user on the schema model. This was leftover and should have been deleted previously